src/components/profile/ProfileSection.vue

```diff
@@ -48,6 +49,16 @@ export default {
         'profile__section--disabled': !this.editing && this.empty,
       };
     },
+    emptyMessageText() {
+      if (
+        !this.userOnOwnProfile ||
+        (this.$route.query.pa !== undefined &&
+          this.$route.query.pa !== DISPLAY_LEVELS.private.value)
+      ) {
+        return this.message;
+      }
+      return this.messageOwn;
+    },
```

Comment:
Apologies, this is one of those things that bugs me while I'm brushing my teeth in the morning.

My confusion stemmed from all the negatives, and the need to "cast" an undefined value into private.

I think the positive formulation of this is much more understandable:
```javascript
if (
  this.userOnOwnProfile &&
  DISPLAY_LEVELS.private.value == (this.$route.query.pa || DISPLAY_LEVELS.private.value)
) {
  return this.messageOwn;
}
return this.message;
```